src/content/docs/cheerpj3/12-reference/00-cheerpjInit.md

```diff
@@ -226,6 +226,68 @@ Example usage:
 cheerpjInit({ javaProperties: ["prop1=value1", "prop2=value2"] });
 ```
 
+### `tailscaleControlUrl`
+
+```ts
+tailscaleControlUrl?: string;
+```
+
+This option expects a string URL of the Tailscale control plane. The control plane verifies the user's identity and validates various keys among the connected peers in the network. This option is used in the scenario of self-hosting, if omitted it will point to the Tailscale control plane.
```

Comment:
```suggestion
This option expects a string URL of the Tailscale control plane. The control plane verifies the user's identity and validates various keys among the connected peers in the network. Only pass this option if you are [self-hosting Tailscale](https://github.com/leaningtech/headscale), if omitted, it will point to the Tailscale control plane.
```